apps/rewards/app/components/Content/Overview.js

```diff
@@ -123,10 +122,13 @@ const RewardsTableNarrow = ({ title, tokens, rewards, fourthColumn, fourthColumn
       <NarrowListReward onClick={generateOpenDetails(reward, openDetails)} key={i}>
         <div style={{ marginTop: '5px', marginRight: '10px' }}>
           <RewardDescription>
-            {reward.description}
+            {e2eTag(reward.description, 'reward-description', i)}
```

Comment:
It'd be better if we could find a less overt way to tag this stuff for testing in our app code, maybe by adding a class or an ID instead. Cypress has tons of tools for digging through the DOM tree so we shouldn't need to do something like this